astropy/table/table.py

```diff
@@ -1000,6 +1000,60 @@ def pprint(self, max_lines=None, max_width=None, show_name=True,
             else:
                 print(line)
 
+    def pprint_all(self, max_lines=-1, max_width=-1, show_name=True,
+               show_unit=None, show_dtype=False, align=None):
+        """Print a formatted string representation of the entire table.
+
+        If no value of ``max_lines`` is supplied then the height of the
```

Comment:
This is no longer quite correct.  I think you can just make it simpler:
```
This method is the same as `astropy.table.Table.pprint` except that
the default ``max_lines`` and ``max_width`` are both both -1 so that
the entire table is printed instead of restricting to the size of the 
screen terminal.
```